docs/ja/forms.md

```diff
@@ -21,7 +21,7 @@ computed: {
   })
 },
 methods: {
-  updateMessage: ({ dispatch }, e) => {
+  updateMessage: (e) => {
```

Comment:
Thanks for correcting this!
Would you mind also change this to a shothand method style definition? The arrow function is still invalid here because we cannot use `this` as a component instance in it.

Reply:
Thank you for reviewing.
I fixed it.